src/Components/Search/Previews/Grids/ArtistSearch/RelatedArtworks.tsx

```diff
@@ -97,7 +97,7 @@ export const RelatedArtworksPreviewFragmentContainer = createFragmentContainer(
       <Responsive>
         {({ xs, sm, md }) => {
           return (
-            <Subscribe to={[SearchBarState]}>
+            <SearchBarConsumer>
```

Comment:
If possible (not sure how much effort it would be to move out of a Class, so not totally critical) this could use a hook. If it's a pain to refactor then all good 👍but if it seems painless then it should also apply to `Responsive`, similar to the comment above.